tools/webhook-apicoverage/resourcetree/resourcetree.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcetree
+
+import (
+	"reflect"
+)
+
+//ResourceTree encapsulates a tree corresponding to a resource type.
+type ResourceTree struct {
+	ResourceName string
+	Root INode
+	forest *ResourceForest
+}
+
+func (r *ResourceTree) createNode(field string, parent INode, t reflect.Type) INode {
+	var n INode
+	switch t.Kind() {
+	case reflect.Struct:
```

Comment:
nit: bad identation

Reply:
I ran gofmt and examples in https://gobyexample.com/switch are also using the same formatting. Let me know if its actually not the "switch" and following "case" that you are referring to.